pkg/datapath/linux/config/config.go

```diff
@@ -77,20 +74,24 @@ var (
 // HeaderfileWriter is a wrapper type which implements datapath.ConfigWriter.
 // It manages writing of configuration of datapath program headerfiles.
 type HeaderfileWriter struct {
+	log                logrus.FieldLogger
 	nodeExtraDefines   dpdef.Map
 	nodeExtraDefineFns []dpdef.Fn
```

Comment:
Why not use an extra define fn instead of adding the bandwith manager to the headerfile writer directly?